meta-zarhus-distro/recipes-tools/zarhus-memtools/zarhus-memtools_1.0.0.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "Tools for measuring and analyzing memory usage of a process"
+HOMEPAGE = "https://docs.zarhus.com"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=5a6917ac6c34ca9b935f702e32987df1"
+
+SRC_URI = "\
+    file://mem_usage \
+    file://mem_test \
+    file://plot_mem_usage.py \
```

Comment:
`mem_test` uses `plot_mem_usage.py`. Does matplotlib work when built without weston or any graphical environment at all? If yes then that's ok if not then you should probably split this into e.g. packageconfig and enable it if `meta-zarhus-webkit` is added

Reply:
matplotlib works without weston (tested). I added `python-matplotlib-3` to the debug packagegroup 4ae28ab1e66a10747747ec96c71725484b3b8513